.github/workflows/ci.yml

```diff
@@ -1114,3 +1114,154 @@ jobs:
+      matrix:
+        php:
+          - '7.3'
+        symfony:
+          - '4.4'
+      fail-fast: false
+    timeout-minutes: 20
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Install system packages
+        run: |
+          apk add \
+            jq \
+            moreutils \
+            unzip \
+      - name: Install mongodb PHP extension
+        run: |
+          apk add $PHPIZE_DEPS
+          pecl install mongodb-$EXT_MONGODB_VERSION
+          docker-php-ext-enable mongodb
+      - name: Disable PHP memory limit
+        run: echo 'memory_limit=-1' >> /usr/local/etc/php/php.ini
+      - name: Install Composer
+        run: wget -qO - https://raw.githubusercontent.com/composer/getcomposer.org/$COMPOSER_INSTALLER_COMMIT/web/installer | php -- --install-dir=/usr/local/bin --filename=composer --quiet
+      - name: Install Symfony Flex
+        run: |
+          composer global require --prefer-dist --no-progress --no-suggest --ansi \
+            symfony/flex
+      - name: Allow unstable project dependencies
+        run: |
+          jq '. + {"minimum-stability": "dev"}' composer.json | sponge composer.json
+      - name: Update project dependencies
+        run: |
+          mkdir -p /tmp/api-platform/core/vendor
+          ln -s /tmp/api-platform/core/vendor vendor
+          composer update --no-progress --no-suggest --ansi
+      - name: Flag held back Symfony packages
+        env:
+          symfony_version: ${{ matrix.symfony }}
+        run: |
+          version_pattern=$symfony_version.x-dev
+          if [ "${symfony_version%.4}" != "$symfony_version" ]; then
+            current_major=${symfony_version%.4}
+            next_major=$((current_major + 1))
+            version_pattern=$version_pattern'|'$next_major.0.x-dev'|'dev-master
+          fi
+          version_pattern=$(echo "$version_pattern" | sed -r 's/\./\\./g')
+          symfony_packages=$(composer show symfony/* | tr -s ' ' '\t' | cut -f1-2 | grep -vE 'polyfill|contracts|mercure')
+          ! echo "$symfony_packages" | grep -vE "$version_pattern"
+      - name: Enable legacy integrations
+        if: startsWith(matrix.php, '7.3')
+        run: echo '::set-env name=LEGACY::1'
+      - name: Clear test app cache
+        run: |
+          mkdir -p /tmp/api-platform/core/var
+          ln -s /tmp/api-platform/core/var tests/Fixtures/app/var
+          tests/Fixtures/app/console cache:clear --ansi
+      - name: Run Behat tests
+        run: |
+          mkdir -p build/logs/behat
+          vendor/bin/behat --format=progress --out=std --format=junit --out=build/logs/behat/junit --profile="$(if [ "$LEGACY" = '1' ]; then echo 'default-legacy'; else echo 'default'; fi)" --no-interaction --colors
+      - name: Upload test artifacts
+        if: always()
+        uses: actions/upload-artifact@v1
+        with:
+          name: behat-logs-php${{ matrix.php }}-symfony${{ matrix.symfony }}
+          path: build/logs/behat
```

Comment:
1267 lines to configure a CI :'(

Reply:
We will be able to fix this soon once support for YAML anchors has landed. :smile: